operations.go

```diff
@@ -266,7 +279,10 @@ func filteredEarlyExitSingle(items Items, column string, operations GroupedOpera
 	}
 	results := []string{}
 	for k := range filteredItemsSet {
-		results = append(results, k)
+		//empty keys crashes frontend.
+		if len(k) > 0 {
+			results = append(results, k)
+		}
```

Comment:
Let's add comment here that this should be removed when the frontend doesn't break.

Reply:
applied